### ISSUE TITLE
Wrap Store throwing error.  "No matching Signature"

### ISSUE THREAD

**Issue by jbarzegar** [tshaddix/webext-redux#212]
Hi there!

My team recently updated `webext-redux` to `2.1.1` and noticed an error was being thrown as soon as you'd wrap the store.

```
wrapStore.js:196 Uncaught (in promise) TypeError: Error in invocation of tabs.query(object queryInfo, function callback): No matching signature.
    at _default (wrapStore.js:196)
```

Digging through `wrapStore` it seemed like it was related to the third argument below
https://github.com/tshaddix/webext-redux/blob/master/src/wrap-store/wrapStore.js#L170

The error itself doesn't seem to be actively breaking anything. But was kind of a surprise

**Comment by tshaddix** [comment]
Yikes! Thanks for letting me know. Not sure how that slipped through the cracks. I must have removed that argument after the smoke test to pass `eslint` rules.

I'll pass something over right now.

**Comment by jbarzegar** [comment]
Thanks @tshaddix ! Really appreciate your quick response

**Comment by tshaddix** [comment]
Should be fixed in 2.1.2! Give me one sec to publish. No problem - completely my bad. I'm embarrassed that it happened.

**Comment by tshaddix** [comment]
Done!
